charts/astronomer/templates/houston/houston-configmap.yaml

```
@@ -120,6 +120,9 @@ data:
         {{- if .Values.global.loggingSidecar.indexPattern }}
         indexPattern: {{ .Values.global.loggingSidecar.indexPattern | squote }}
         {{- end }}
+        {{- if .Values.global.loggingSidecar.indexNamePrefix }}
+        indexNamePrefix: {{ .Values.global.loggingSidecar.indexNamePrefix }}
```

Should we do this as something that works for both fluentd and vector?
```suggestion
        indexNamePrefix: {{ .Values.global.logging.indexNamePrefix }}
```